src/elm/Page/Community.elm

```diff
@@ -173,7 +173,17 @@ view loggedIn model =
                                 ]
 
                           else
-                            text ""
+                              div [ class "bg-white py-6 sm:py-8 px-3 sm:px-6 rounded-lg mt-4" ]
+                                 (Page.viewTitle (t "community.objectives.title_plural")
+                                     :: List.indexedMap (viewObjective loggedIn model community)
+                                         community.objectives
+                                     ++ [ if canEdit then
+                                             viewObjectiveNew loggedIn editStatus community.symbol
+
+                                           else
+                                             text ""
+                                        ]
+                                 )
```

Comment:
The `else` branch is the same as `then`. And the `div [ class "container mx-auto px-4" ]`, like in the `then` branch, is also important here. It makes the block to be rounded and restricts the `width` of the block.

It looks like the cause of the problem is the `canEdit` predicate. I guess it's better to just remove it and don't duplicate the code:

```elm
                        , if community.hasObjectives then
                            div [ class "container mx-auto px-4" ]
                                [ div [ class "bg-white py-6 sm:py-8 px-3 sm:px-6 rounded-lg mt-4" ]
                                    (Page.viewTitle (t "community.objectives.title_plural")
                                        :: List.indexedMap (viewObjective loggedIn model community)
                                            community.objectives
                                        ++ [ if canEdit then
                                                viewObjectiveNew loggedIn editStatus community.symbol

                                             else
                                                text ""
                                           ]
                                    )
                                ]

                          else
                            text ""
                        ]
```

@RodrigoLeiteF Please, have a look!

Reply:
You are right @amiskov, I missed the point here